rosbag2_transport/test/rosbag2_transport/test_play.cpp

```diff
@@ -277,25 +298,89 @@ TEST_F(RosBag2PlayQosOverrideTestFixture, topic_qos_profiles_overridden_incompat
   // which should not be a compatible offer and therefore we should receive no messages.
   const auto qos_request = rclcpp::QoS{rclcpp::KeepAll()}.reliable();
   const auto qos_playback_override = rclcpp::QoS{rclcpp::KeepAll()}.best_effort();
-
   const auto topic_qos_profile_overrides = std::unordered_map<std::string, rclcpp::QoS>{
     std::pair<std::string, rclcpp::QoS>{topic_name_, qos_playback_override},
   };
-  play_options_.topic_qos_profile_overrides = topic_qos_profile_overrides;
+  // If any messages were going to come in, it should happen in under 1s even in slow scenarios
+  const auto timeout = 3s;
```

Comment:
Any reason for 3 and not 5 in other places?

Reply:
lol you defined this - i've just moved it to the top of the function.

However, I'd say the reasoning is because this one _always_ times out, so we are a little less patient and fail earlier.

Reply:
That's fine, I was wondering if there's any specific reason for a longer time out in the other tests :)